specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2021-11-01-preview/managedClusters.json

```diff
@@ -2650,6 +2653,11 @@
         "workloadRuntime": {
           "$ref": "#/definitions/WorkloadRuntime"
         },
+        "messageOfTheDay": {
+          "type": "string",
+          "title": "Message of the day for Linux nodes, base64-encoded.",
+          "description": "A base64-encoded string which will be written to /etc/motd after decoding. This allows customization of the message of the day for linux nodes. It must be a static string (i.e., will be printed raw and not be executed as a script)."
```

Comment:
Assume this is just ignored for Windows node pools? Or is specifying it on a Windows node pool an error?

Reply:
I initially ignored it, but that felt a bit misleading, so I ended up adding a validation error. welcome a review on the RP side ;)

Reply:
+1 to erroring - silently ignoring fields is a smell IMO.

Might be worth tacking "This field cannot be set on Windows node pools" or something onto the end of the description, although that is _kindof_ implied based on the "Message for Linux nodes" so feel free to ignore this if you dont feel its worth the extra text

Reply:
added a clarifying comment